cmake/FindH5Part.cmake

```diff
@@ -1,32 +1,48 @@
-#------------------------------------------------------------------------------#
-# Copyright (c) 2016 Los Alamos National Security, LLC
-# All rights reserved.
-#------------------------------------------------------------------------------#
+################################################################################
+#
+# \file      cmake/FindH5Part.cmake
+# \author    C. Junghans
+# \copyright 2016, Los Alamos National Security, LLC.
+# \brief     Find the H5Part library
+# \date      Mon 05 Dec 2016 09:02:53 AM MST
+#
+################################################################################
 
-# - Find H5Part
-# Find the native H5Part headers and libraries.
+# Find H5Part headers and libraries
+#
+#  H5PART_FOUND        - True if H5Part is found
+#  H5PART_INCLUDE_DIR  - H5Part include files paths
+#  H5PART_LIBRARY      - List of H5Part libraries
+#
+#  Set H5PART_ROOT before calling find_package to a path to add an additional
+#  search path, e.g.,
 #
-#  H5Part_INCLUDE_DIRS - where to find flecsi.h, etc.
-#  H5Part_LIBRARIES    - List of libraries when using flecsi.
-#  H5Part_FOUND        - True if flecsi found.
+#  Usage:
+#
+#  set(H5PART_ROOT "/path/to/custom/h5part") # prefer over system
```

Comment:
That would force cmake to try to find every library at `/path/to/custom/h5part`. This way one can customize per lib.